src/iop/borders.c

```diff
@@ -220,10 +220,13 @@ int distort_transform(dt_iop_module_t *self, dt_dev_pixelpipe_iop_t *piece, floa
   const int border_size_t = border_tot_height * d->pos_v;
   const int border_size_l = border_tot_width * d->pos_h;
 
+  // nothing to be done if parameters are set to neutral values (no top/left border)
+  if (border_size_l == 0 && border_size_t == 0) return 1;
+
 #ifdef _OPENMP
 #pragma omp parallel for default(none) \
   dt_omp_firstprivate(points, points_count, border_size_l, border_size_t)  \
-  schedule(static)
+  schedule(static) if(points_count > 100)
```

Comment:
This seems really wrong to me...

If OpenMP is active and we have less than 100 points we do nothing!

Reply:
No, the "if" on the OpenMP directive means only parallelize if the condition is true -- if it's false, run single-threaded.

OpenMP 4.5 specifications, section 2.5.1 (pages 50-51):
```
if (IfClauseValue = false)
   then number of threads = 1;
```

Reply:
Oh ! That's part of the OpenMP directive, I was completely confused when seeing that. Sorry!